### PR TITLE
Make Service Partner optional in Info

### LINKED ISSUE #63: Field "service_partner" of type ServicePartner is missing in Info instance
I hadn't selected a Service Partner yet. It should be optional in the info instance.
I've made a PR, not sure if the changes are correct :) 
https://github.com/skodaconnect/myskoda/pull/62

### PR DESCRIPTION
**Pull request by maartenla**
If you haven't chosen a Service Partner yet in the app this field is not sent from the API.

If something isn't correct let me know!

@dvx76 

Edit: fixes #63 and skodaconnect/homeassistant-myskoda#50

### CODE REVIEW

File: myskoda/models/info.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ class Info(DataClassORJSONMixin):
     name: str
     capabilities: Capabilities
     device_platform: str = field(metadata=field_options(alias="devicePlatform"))
-    service_partner: ServicePartner = field(metadata=field_options(alias="servicePartner"))
+    service_partner: ServicePartner | None = field(default=None, metadata=field_options(alias="servicePartner"))
     workshop_mode_enabled: bool = field(metadata=field_options(alias="workshopModeEnabled"))
```
Comment (dvx76) [inline myskoda/models/info.py]: The linter trips over line lengths here and there's also some requirement about default fields coming after all non-default fields. Can you change it to this:

```suggestion
    workshop_mode_enabled: bool = field(metadata=field_options(alias="workshopModeEnabled"))
    service_partner: ServicePartner | None = field(
        default=None, metadata=field_options(alias="servicePartner")
    )
```

tip: auto-formatting with ruff (https://docs.astral.sh/ruff/) handles this.

**Review by dvx76 — CHANGES_REQUESTED** [review]
Thanks for your contribution! There's some minor formatting to change to satisfy the linter. Let me know if it's a problem to update it.

**Review by Prior99 — DISMISSED** [review]

File: myskoda/models/info.py (hunk 1 above)
Comment (maartenla) [reply myskoda/models/info.py]: Fixed the ordering of the fields

**Review by WebSpider — APPROVED** [review]
lgtm

**Comment by WebSpider** [comment]
Thanks for your contribution!

**Pull request merged**